media/css/m24/flag.scss

```diff
@@ -43,10 +39,10 @@
 
 .m24-c-flag-button {
     $font-size: 0.75;
-    background-color: $m24-color-dark-mid-gray;
-    border-color: transparent; // High Contrast Mode support
+    background-color: $m24-color-alt-white;
```

Comment:
In Figma, this is transparent: https://www.figma.com/design/umfT9uNYJwAjIwgz360BdB/MVP-3.0-Mozilla.org?node-id=2560-190615&m=dev

But I thought with a white text it might be best to specify the background color for contrast purposes. Not sure if that's the right call.